pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -1,22 +1,36 @@
-{ stdenv, lib, fetchurl, python2Packages, gettext }:
+{ stdenv, lib, fetchFromGitHub, python2Packages, gettext }:
 
 python2Packages.buildPythonApplication rec {
   name = "LinkChecker-${version}";
-  version = "9.3";
+  version = "9.3.1";
 
   buildInputs = with python2Packages ; [ pytest ];
-  propagatedBuildInputs = with python2Packages ; [ requests ] ++ [ gettext ];
+  propagatedBuildInputs = with python2Packages;[ 
+    # pin requests version until next release.
+    # see: https://github.com/linkcheck/linkchecker/issues/76
+    (requests.overridePythonAttrs(old: rec {
```

Comment:
Please have a look at https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/home-assistant/default.nix for how to pin dependency version.